src/css/components/_captions-settings.scss

```diff
@@ -62,6 +60,11 @@
   display: inline-block;
 }
 
+// style the second select for text colors
+.vjs-text-track-settings fieldset span > select {
+  max-width: 7.3em;
```

Comment:
At 7.3em the selects won't ever collapse, so that we can ensure that we always have the right `min-height` so that the `controls` never touch the bottom of the modal.